src/hooks/api/useInvestments.ts

```diff
@@ -53,7 +56,7 @@ export function useInvestments<TData = InvestmentAccount[]>(
     queryKey,
     queryFn: fetcher(
       () => getInvestments(
-        (accounts as Account[]).filter(a => a.type === 'INVESTMENT'),
+        (accounts as Account[]).filter(a => a.type === 'INVESTMENT' && !a.placeholder),
```

Comment:
We still calculate for ALL investments at once for now. The way we filter is by using the `select` function